src/main/java/spireMapOverhaul/zones/volatileGrounds/monsters/GremlinArchmage.java

```diff
@@ -0,0 +1,183 @@
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.localization.MonsterStrings;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.monsters.beyond.Exploder;
+import com.megacrit.cardcrawl.monsters.beyond.SnakeDagger;
+import com.megacrit.cardcrawl.powers.*;
+import spireMapOverhaul.SpireAnniversary6Mod;
+import spireMapOverhaul.zones.volatileGrounds.powers.ExplodePlusPower;
+
+import java.util.Iterator;
+
+public class GremlinArchmage extends CustomMonster {
+    public static final String ID = SpireAnniversary6Mod.makeID("GremlinArchmage");
+    private static final MonsterStrings monsterStrings = CardCrawlGame.languagePack.getMonsterStrings(ID);
+    public static final String NAME = monsterStrings.NAME;
+    public static final String[] MOVES = monsterStrings.MOVES;
+    private static final String TEMP_IMG = SpireAnniversary6Mod.makeImagePath("monsters/VolatileGrounds/GremlinArchmage/GremlinArchmage.png");
+    //TODO: ADD ANIMATION
+    //private static final String IMG = SpireAnniversary6Mod.makeImagePath("monsters/VolatileGrounds/GremlinArchmage/skeleton.png");
+    //private static final String ATLAS = SpireAnniversary6Mod.makeImagePath("monsters/VolatileGrounds/GremlinArchmage/skeleton.atlas");
+    //private static final String SKELETON = SpireAnniversary6Mod.makeImagePath("monsters/VolatileGrounds/GremlinArchmage/skeleton.json");
+    private static final byte SUMMON = 0;
+    private static final byte ATTACK = 1;
+    private static final byte BLOCK = 2;
+    private static final int ATTACK_DAMAGE_1 = 12;
+    private static final int A2_ATTACK_DAMAGE_1 = 14;
+    private static final int BLOCK_AMOUNT = 8;
+    private static final int A17_BLOCK_AMOUNT = 12;
+    private static final int HP_MIN = 95;
+    private static final int HP_MAX = 98;
+    private static final int A7_HP_MIN = 110;
+    private static final int A7_HP_MAX = 115;
+    private int damage1;
+    private int blockAmt;
+    public AbstractMonster[]  exploders = new AbstractMonster[2];
+    public static final float[] POSX;
+    public boolean firstTurn = true;
+    static
+    {
+        POSX = new float[]{-430, -200.0F};
+    }
+    
+    public GremlinArchmage(final float x, final float y) {
+        super(NAME, ID, HP_MAX, -5.0F, 0, 120.0f, 220f, TEMP_IMG, x, y);
+        if (AbstractDungeon.ascensionLevel >= 7) {
+            this.setHp(A7_HP_MIN, A7_HP_MAX);
+        } else {
+            this.setHp(HP_MIN, HP_MAX);
+        }
+        if(AbstractDungeon.ascensionLevel <= 3) {
+            this.damage.add(new DamageInfo(this, ATTACK_DAMAGE_1));
+            damage1 = A2_ATTACK_DAMAGE_1;
+        }
+        else
+        {
+            this.damage.add(new DamageInfo(this, A2_ATTACK_DAMAGE_1));
+            damage1 = ATTACK_DAMAGE_1;
+        }
+        if(AbstractDungeon.ascensionLevel >= 17)
```

Comment:
Usually the A17 buff for normal enemies is a bit stronger than getting more block. For example, Orb Guardian gains strength faster, Transient gets an extra turn, Darklings get strength gain added to the block move, Spire Growth applies more constricted, etc.

Looking over what this enemy does, the suggestion that immediately jumps out at me is to have the exploders spawn with higher HP.

Another idea that might be more interesting is to give each summoned Exploder 3 or 4 more strength than the previous ones... that actually seems like a pretty neat way to add a bit of scaling to the fight. The initial set would have 0 strength, but the first summon would have 4 strength, then 8 for the second summon, etc. Another twist on that could be to have that mechanic at lower ascensions too, but with a much lower amount (e.g. 1 or 2) and have A17 increase it to 3 or 4. Or combine this with my other suggestion and have their HP increase as well.

Reply:
I could make the exploders explode one turn earlier

Reply:
Actually no, would require some dodgy code. I'll just make it add some Strength.